configure.ac

```
@@ -188,6 +188,11 @@ AC_SUBST([WERROR_DCFLAG])
 dnl Check for required modules: curl and sqlite at the moment
 PKG_CHECK_MODULES([curl],[libcurl])
 PKG_CHECK_MODULES([sqlite],[sqlite3])
+PKG_CHECK_MODULES([dbus], [dbus-1 >= 1.0],
+  [AC_DEFINE([HAVE_DBUS], [1], [Define if you have dbus-1])]
+  ,
+  [AC_MSG_ERROR([dbus-1 development files not found. Please install libdbus-1-dev (Debian) or dbus-devel (Red Hat)])]
+)
```

```suggestion
PKG_CHECK_MODULES([dbus], [dbus-1 >= 1.0])
```

Any reason not to simply call `PKG_CHECK_MODULES` and then use `dbus_LIBS` in `Makefile.in` like it is done for `curl` and `sqlite`?

---

Because the code is in flux.

When I wrote the original, I was forcing use of library. I added only today the detection stuff.

---

Resolved with https://github.com/abraunegg/onedrive/pull/3274/commits/41066a35e72e169630ab2fa3f49170b8233eaa04